frontend/src/lib/OutputArtifactLoader.ts

```diff
@@ -450,45 +495,37 @@ async function getArtifactTypes(): Promise<ArtifactType[]> {
   return res.getArtifactTypesList();
 }
 
-function filterTfdvArtifactsPaths(artifactTypes: ArtifactType[], artifacts: Artifact[]): string[] {
-  const tfdvArtifactTypeIds = artifactTypes
-    .filter(artifactType => artifactType.getName() === 'ExampleStatistics')
+function filterArtifactUrisByType(
+  artifactTypeName: string,
+  artifactTypes: ArtifactType[],
+  artifacts: Artifact[],
+): string[] {
+  const artifactTypeIds = artifactTypes
+    .filter(artifactType => artifactType.getName() === artifactTypeName)
     .map(artifactType => artifactType.getId());
-  const tfdvArtifacts = artifacts.filter(artifact =>
-    tfdvArtifactTypeIds.includes(artifact.getTypeId()),
+  const matchingArtifacts = artifacts.filter(artifact =>
+    artifactTypeIds.includes(artifact.getTypeId()),
   );
 
-  const tfdvArtifactsPaths = tfdvArtifacts
-    .filter(artifact => artifact.getUri()) // uri not empty
-    .flatMap(artifact => [
-      artifact.getUri() + '/eval/stats_tfrecord', // eval uri
-      artifact.getUri() + '/train/stats_tfrecord', // train uri
-    ]);
+  const tfdvArtifactsPaths = matchingArtifacts.map(artifact => artifact.getUri()); // uri not empty
```

Comment:
To guarantee only return non empty uris, you still need the filter step.
e.g.
```
const tfdvArtifactsPaths = matchingArtifacts
  .map(artifact => artifact.getUri())
  .filter(uri => uri);
```

Reply:
Done.